extensions/wikia/EmbeddableDiscussions/EmbeddableDiscussionsController.class.php

```diff
@@ -52,8 +110,6 @@ public static function render( $input, array $args ) {
 			$columns = self::COLUMNS_MAX;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)